keras_cv/layers/preprocessing/random_brightness.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+_SCALE_VALIDATION_ERROR = (
+    "The `scale` should be number or a list of two numbers "
```

Comment:
"The `scale` argument should be a number (or a list of two numbers) "
"in the range [-1.0, 1.0]. "